pkg/agent/testdata/AKSUbuntu2204+ManagedGPUExperienceAFEC+Disabled/CustomData

```diff
@@ -67,7 +67,7 @@ write_files:
3AgjUuNmsB0Ws8bXAQXtbeGGF+Bo04A+4y2SmhzDpdE3zOFI6U+G9x4ike0nTqt56cGy+GQ0KzyRKpbklfpumN4eddCwHgbHVUtc1c3DEGbDfkgmKI9y/5coj3DfRwhBF//8qVu7NbGdr46E8lY6szfw50+FffQgHJQDZYJU3TwIyx18KiG5Ut8148i28H8x44jvOU+FHZxFPfHQsDZTQfHIW4Zo30ygdNv7+VNf3Lxxv7iQ+7mLr3C/kt3VlGZjvFwsx7xGaLEwzdzgeQ/5HI6KSte5qx6fbebXwd+ut9GuMQp3S09U3dlbbawp1ljX3ibxnmdVD9NqHHUnt1fgZMCU/LiKCZC6I1O30rP9ZGKNpuPDOgTrbZRU9u3DgKAntqf38G1iZc3F0A+7/1VMs0eQ1NVCqQzQjTu7XCwBL936LTtkeoFYhbRod6Jj5KQbRdtTwRZZmbmANuwYHjifMRv7krG4I+HzrsdNUksyEc24y/k0Y0VuU7OlGYb+nSeUIGf2Gg60oKc5TGDgwGnowrpgixCD7twAAlUgCAnA0TYO77YJILceZnXrelnSRXExFQKpT/Jh0k2GB164V06Ly8kRkJ2Cwa9hwutxfzcYJMjxxvsCCF++Ar4bO8faYOsc7Evh5hM+gcbTxveCOzNUfH8M7z0Hzqg+0p6x2zvP94FsaJdvlguximkt7cm+IRh04gBLWL9tHnlY0aevT495SbuTPYaovYayGnn4+uskPGgbxVsFoIsFuizLoyiPB03Qw63n3AqyUrx5c6pZumosJ6ZYS6pIel7cjssLrj2aLqNXNUwWg1FEmtzWOdXoWMylkWzD8YRkL+p24thhnREU+5yDVajPp8DDwAkRirYEulVu5vJmrM0u5pY+n1qjua4vF6Y6rncuHrqNV0WIZSLwECKW4rbygsw7ktpWv6zUpneSceNLRr6lXVgzVR3HlWRYjaTDvFcZob3SzlqS7TJbWiMvL2cJlJ+JYnagctSaksIyl4BU53YobgjfDaTWm3n+PgrjTdkBNebvWG/hFmB8m0eeFD8zjDf7Grr1LcVCDobxZhHd3MEnJSK3MUG8qGlSf27LXlt2RG6HUvdESgEwvrUinAPoCHX4K9rXlTaqwDSUglASi69Xwjzx69+ZdHCiDONNUtk4vdP+lv7vWkkocQJqThcdQP8eSt2jzR2Bm+0xi9K0H+6AfA9ciD0E3aHUrehbAi94ToR6qc3AM/ACa2MT53Y4+BEguPVtB7r0Zy/AkKouwwHgCnn/n1MKd/T+/Rne2g48+/DhU/e4H0sCY9/RVwkGpsAlKI7BM2AWKiBlUyYkNEoJxUKvJyA5BqQEnvzIsfwIAvjI1TUOc/RVAnNM++3/UyDxm6oehfd/zQbQksSUSSclQpJh7uIouDHlbwNbWvXIsL2gX6RMQnimkzldsGln14CwlSOzepD8DV1Gz3GaHH/yo7Cs4t13sTy/Ut9ZytJ8Q1fYj2CXIkwOFfIGvD49BXLIruYC8hrEt3OxTvJkdUCuw4w44SJ0qm6k67z0lrvyy9tANb2CVb4UdpFZ5LP0i/QKzMYo5vporuzsvT8zpqqvsr91b4NC9EfpIsiK2I/xzLCM5cWF9mtdGpKzN7jNiTAJN4BfAd6QZ1NN9J7w6rtchFqPnyv7WdB1aS5jl1FCIT7rAJA4jljh7/KM8VmimwILS8Fn8SBkIH3Ty4qBS1WPnaDysQsrH0dYfHzoDPEyzzyhlOXbJ9c4nwHpZLCJE88F5mVjTYbaS/iZMZI1Qmth1CUjE7NB/4bDQMgi+GJRRm3FLK79DTwc+jY9KOcK7/6vyLURuVJxg38HmUpAZBnFFYriCv/lOzjOugfIWlPKS7O4sVIKMVMAhoR4gVh683/lrEHO/j32J34MqUp2z86lGxfPI7KNSHZBAZDfgPR2AnaqlOUg3KLw8QlI30hAljf2o8xulHj1R1xbAF4KIlNzpVZh+wzYyBTXRXh4JFx7sL0/pQ+TAiTdbLgHX28A+n0gScIlB+ntBryPOJxboOSvwyPhdY6q162pev3ZVL1upup1gaoYBK2H3SPtwhh+C35Mg9OE5u+/+bCTjkFFmbgUQdOVwAqvASszL6fsbYdZM0DCujK/VUWz89kLVQil6rJNk/lImVCFYjTX1aSwa2VqEzs9ukH2A3Ml84sOudF7Ql+MZ0bezvMV1+ifyz1llRC/ogutFqZSO0ioSIqzByEBNzCptdyTmqv+V+PItd8b3gvS5oUQY46cA2dAPOA4JSwmKy4UHw9btJl06lHhyHFY9d+eVJrDz53CFIkx0UaqUOG3ECObNsU+C4bmrFMFAQiqbovPnXNKBJdPNhUgFYGkdVDJUk3fX6ozVVdMdSxAloNKK9Dlz5H5ier8n33jS3nZMEABpmmEJbCkeGsVRL5c6ximDt1hKkMGnb7OmH+ax3ALAxcGjgfxMAiTAq9sinGP2GgNSUeHHyMPQTzkksAFoKOsCET5R+8Z7g+dKdyE6GlqPw4FKqfqdK6/sybaVDN3ndFi+Y8oJLYIMVosk9dpeleL9doUNS2s1KYJbb3imJ2Bu4h4/I+rPm7j8O/yDc5Tdr9k/dWcia9T5RE27J5na/5W1XVtrMYF8SpWYmyTdZlfSt760doL8veY5hbg895uypnP+5vEsfEWJsgmcP3UqlxPLj6ecoeq76Z6+W4os0D7BBnAXrCmm1N6a+pQ+LEfYcS3I9FFk2MH6IrYc9UUMrORa8NNGMjctpTbfesM6fyy16oO95vQ8/RV29Fz7C76ytoR5aw3LWhx1psyCcmkjy6nQrfsV/XXxVw3Vf0A6XTWG5lfNgNRk3TW40/q8MQy2QDILtC+w3Jyw16L+3gNYgeujdxY7Y8TctLHKjt0Jk/rxZAKYG6o7BpmVu1LfJrU/pKdMKKKP+45+B7Ish3rr9LZyQ+ng4FUUeW6IKaHiUJKQDuZyEbRKBzpJCRZP3+hZP/fAAAA//9a3DTX1q0AAA==
```

Comment:
why does it need to update the testdata when you are just updating the components.json?